corehq/toggles.py

```diff
@@ -1453,3 +1453,10 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_CUSTOM,
     namespaces=[NAMESPACE_OTHER],
 )
+
+USE_XPATH_EXPRESSION = StaticToggle(
```

Comment:
More descriptive name would be nice - maybe something that includes "case list".